### PR TITLE
chore: IE11 smoke tests

### PR DESCRIPTION
**Pull request by markov00**
## Summary

This PR add a simple smoke test for IE11.
The test runs in a GH action, attaching a selenium driver/ webDriver to IE11 and checking if at least one `echChart` element is present on the page of the current Playground.

The test is very basic but helps at the moment to check if we have added an incompatible IE11 library that prevent the chart from rendering (usually because the bundle can't be evaluated by IE11).

In this PR, the action is run on the fork repo (see https://github.com/markov00/elastic-charts/actions?query=workflow%3A%22Smoke+Tests%22)
I'm pretty sure that, after we merged this PR, the Action will run from the elastic repo

For reference: https://dev.to/mizchi/run-crossbrowser-e2e-testing-on-github-actions-671

### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.

- ~[ ] Any consumer-facing exports were added to `src/index.ts` (and stories only import from `../src` except for test data & storybook)~
- [x] This was checked for cross-browser compatibility, [including a check against IE11](https://github.com/elastic/kibana/blob/master/CONTRIBUTING.md#cross-browser-compatibility)
- ~[ ] Proper documentation or storybook story was added for features that require explanation or tutorials~
- [x] Unit tests were updated or added to match the most common scenarios
- [x] Each commit follows the [convention](https://github.com/elastic/elastic-charts/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by robgil** [comment]
# [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=h1) Report
> Merging [#543](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=desc) into [master](https://codecov.io/gh/elastic/elastic-charts/commit/ce4e84fac8b51861cde377303ecaf9038611158b&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #543   +/-   ##
=======================================
  Coverage   75.57%   75.57%           
=======================================
  Files         181      181           
  Lines        5691     5691           
  Branches     1104     1104           
=======================================
  Hits         4301     4301           
  Misses       1374     1374           
  Partials       16       16           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=footer). Last update [1a73e4a...f3736f1](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by robgil** [comment]
# [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=h1) Report
> Merging [#543](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=desc) into [master](https://codecov.io/gh/elastic/elastic-charts/commit/ce4e84fac8b51861cde377303ecaf9038611158b&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #543   +/-   ##
=======================================
  Coverage   75.57%   75.57%           
=======================================
  Files         181      181           
  Lines        5691     5691           
  Branches     1104     1104           
=======================================
  Hits         4301     4301           
  Misses       1374     1374           
  Partials       16       16           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=footer). Last update [1a73e4a...436c6a1](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by robgil** [comment]
# [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=h1) Report
> Merging [#543](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=desc) into [master](https://codecov.io/gh/elastic/elastic-charts/commit/ce4e84fac8b51861cde377303ecaf9038611158b&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #543   +/-   ##
=======================================
  Coverage   75.57%   75.57%           
=======================================
  Files         181      181           
  Lines        5691     5691           
  Branches     1104     1104           
=======================================
  Hits         4301     4301           
  Misses       1374     1374           
  Partials       16       16           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=footer). Last update [1a73e4a...b675beb](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by robgil** [comment]
# [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=h1) Report
> Merging [#543](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=desc) into [master](https://codecov.io/gh/elastic/elastic-charts/commit/ce4e84fac8b51861cde377303ecaf9038611158b&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #543   +/-   ##
=======================================
  Coverage   75.57%   75.57%           
=======================================
  Files         181      181           
  Lines        5691     5691           
  Branches     1104     1104           
=======================================
  Hits         4301     4301           
  Misses       1374     1374           
  Partials       16       16           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=footer). Last update [1a73e4a...f0e7403](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by robgil** [comment]
# [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=h1) Report
> Merging [#543](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=desc) into [master](https://codecov.io/gh/elastic/elastic-charts/commit/ce4e84fac8b51861cde377303ecaf9038611158b&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #543   +/-   ##
=======================================
  Coverage   75.57%   75.57%           
=======================================
  Files         181      181           
  Lines        5691     5691           
  Branches     1104     1104           
=======================================
  Hits         4301     4301           
  Misses       1374     1374           
  Partials       16       16           
```



------

[Continue to review full report at Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=footer). Last update [1a73e4a...e368d70](https://codecov.io/gh/elastic/elastic-charts/pull/543?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

File: browsers/browsers.test.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    if (capabilities) {
+      driver = await new webdriver.Builder().withCapabilities(capabilities).build();
+    }
+  });
+
+  afterAll(async () => {
+    await driver.quit();
+  });
+
+  test('Google', async () => {
```
Comment (nickofthyme) [inline browsers/browsers.test.ts]: Update test naming?

Comment (markov00) [reply browsers/browsers.test.ts]: solved

**Comment by markov00** [comment]
> LGTM, Not sure how to test this locally with IE11.

@nickofthyme I know, I think the only way to test it is using virtualbox and a removeWebDriver: https://stackoverflow.com/questions/12380454/how-to-setup-ie-webdriver-on-a-remote-box
but for now I think the we can just test it on the action and manual testing in IE11 as we are doing already

**Pull request merged**